target/linux/ath79/dts/ar9331_tplink_tl-wr710n-v1.dts

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9331.dtsi"
+
+/ {
+	model = "TP-Link TL-WR710N v1";
+	compatible = "tplink,tl-wr710n-v1", "qca,ar9331";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	gpio-keys-polled {
```

Comment:
"keys" is enough. (And recommended by the [DeviceTree spec](https://github.com/devicetree-org/devicetree-specification/blob/master/source/devicetree-basics.rst#generic-names-recommendation)])

Reply:
Changed